CI/flatpak/com.obsproject.Studio.json

```diff
@@ -236,6 +263,15 @@
             "patch -Np1 -i obs-deps/CI/patches/FFmpeg-4.4.1-libaomenc.patch",
             "patch -Np1 -i obs-deps/CI/patches/FFmpeg-9010.patch"
           ]
+        },
+        {
+          "type": "shell",
+          "only-arches": [
+            "x86_64"
+          ],
+          "commands": [
+            "sed -i 's/enabled libsvtav1         && //g' ./configure # Enable svt-av1 only on x86_64 without creating a second FFmpeg module"
```

Comment:
What prevents libsvtav1 from being included in non-x86_64 builds?

Reply:
SVT-AV1 is only tested on x86_64.  This is probably in part due to CPU requirements and memory requirements.  I don't think we want to support an unsupported use-case.

https://github.com/AOMediaCodec/SVT-AV1#operating-system

Reply:
It does not work on other arch, only on x86_64.

Reply:
So from this page, it seems like it should work in 64 bit systems in general.

I'm not super happy with sprinkling these `only-arches` around. The current CI pipeline currently doesn't build anything other than x86_64, so I don't personally see the point of adding these conditionals when they're unused. I'd rather add aarch64 support in a single, self-contained PR, and not pollute other PRs with these changes. Would you mind removing it?

Reply:
> So from this page, it seems like it should work in 64 bit systems in general.

If you look at the hardware section, I think SVT-AV1 is only meant to be used right now by x86_64 and not any 64 bits arch.

> I'm not super happy with sprinkling these `only-arches` around.

Since I know that SVT-AV1, I just wanted to add the module the right way, avoiding people to build SVT-AV1 on unsupported arches.

> The current CI pipeline currently doesn't build anything other than x86_64, so I don't personally see the point of adding these conditionals when they're unused.

Since there is checks on obs-deps to avoid building SVT-AV1 on non-x86_64, I assumes adding the same here would be a good idea.

> I'd rather add aarch64 support in a single, self-contained PR, and not pollute other PRs with these changes. Would you mind removing it?

There is no aarch64 purpose on how this commit is made, just trying to enable SVT-AV1 on the right arch just like obs-deps do.